src/main/migrations/changesets/initial_schema.xml

```diff
@@ -0,0 +1,581 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Yes, if a little hacky: run on an empty DB with liquibaseUpdate() disabled and adding a line to create the schema using Slick, followed by running Liquibase CLI's **createChangeLog** or **diffChangeLog** as seen [here](https://broadinstitute.atlassian.net/wiki/display/GAWB/Liquibase+for+Rawls)